# Robot parameters 
tcp_speed: 0.05

# Gripper parameters
tcp2fingertip: 0.32 # distance from tcp to gripper fingertip
opening_per_count: 0.00065 # gripper stroke opening per rPr count
finger_thickness: 0.005 
max_opening: 0.1523 # max stroke of gripper excluding finger thickness
gripper_speed: 0.1 # value between 0.013 and 0.100
gripper_force: 10 # value between 0 and 100

# Object dimension
object_thickness: 0.000 #0.01 # object thickness in meters
object_length: 0.087 # object length in meters

# Initial configuration
delta_0: 0.053 #0.0425 # distance from fingertip to object tip within gripper
theta_0: 30.0

# Intermediate configuration
psi_regrasp: 43.0
theta_tilt: 18
tuck: 20

# Action axis
axis: [1, 0, 0]

# Simulation
sim: 0